index.js

```diff
@@ -89,7 +89,7 @@
   if (typeof module !== 'undefined') {
     module.exports = format;
   } else if (typeof define === 'function' && define.amd) {
-    define(format);
+    define(function () { return format; });
```

Comment:
Thanks for the update. Would you mind squashing? I don't like to have failing tests at any point in a repository's history.